pulse_actions/handlers/treeherder_resultset.py

```diff
@@ -54,9 +55,9 @@ def on_resultset_action_event(data, message, dry_run, stage=False):
         else:
             status = 'Dry-mode, no request sent'
     elif action == "trigger_all_talos_jobs":
-        trigger_all_talos_jobs(repo_name, revision, times, dry_run=dry_run)
+        trigger_all_talos_jobs(repo_name, revision, times, priority, dry_run=dry_run)
         if not dry_run:
-            status = 'trigger_all_talos_jobs %s times request sent' % times
+            status = 'trigger_all_talos_jobs {0} times request sent with priority {1}'.format(times, priority)
```

Comment:
Thank you so much for adding it to the output.
Instead of -1 in the output we can say "with priority lower than normal".